src/main/java/core/basesyntax/strategy/impl/PurchaseMovementHandler.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitMovement;
+import core.basesyntax.storage.FruitDao;
+import core.basesyntax.strategy.GoodHandler;
+
+public class PurchaseMovementHandler implements GoodHandler {
+    private final FruitDao fruitDao;
+
+    public PurchaseMovementHandler(FruitDao fruitDao) {
+        this.fruitDao = fruitDao;
+    }
+
+    @Override
+    public void makePosting(FruitMovement movement) {
+        fruitDao.updateBalance(movement.getFruit(),
+                fruitDao.getAmount(movement.getFruit()) - movement.getAmount());
```

Comment:
let's make methods in `fruitDao` that will subtract or add fruit amount
and call them

Reply:
same in other handlers